include/aspect/utilities.h

```diff
@@ -156,6 +156,11 @@ namespace aspect
     bool fexists(const std::string &filename);
 
     /**
+     * Retrieve the last line of a file
+     */
+    std::string get_last_line(std::ifstream *in);
```

Comment:
Well, the use case was to read a file buffer and not a stream of characters. Since this was a specific case, I used the ifstream object as opposed to the istream.